File: src/G2_py/PYA205.py
in1=input()

if ('A' <= in1 <= 'Z') | ('a' <= in1 <= 'z'):
    print("{:} is an alphabet.".format(in1))
elif ('0' <= in1 <= '9'):
    print("{:} is a number.".format(in1))
else:
    print("{:} is a symbol.".format(in1))

File: src/G2_rust/PYA205.rs
use std::io;

fn main(){
    let mut input: String= String::new();
    io::stdin().read_line(&mut input).unwrap();
    let in1: char=input.trim().parse().unwrap();

    if in1.is_alphabetic(){
        println!("{:} is an alphabet.",in1);
    }else if in1.is_numeric(){
        println!("{:} is a number.",in1);
    }else{
        println!("{:} is a symbol.",in1);
    }
    
}